modules/apps/style-book/style-book-api/src/main/java/com/liferay/style/book/exception/ThemeIdNotFoundStyleBookEntryFileException.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.style.book.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Anderson Luiz
+ */
+public class ThemeIdNotFoundStyleBookEntryFileException
+	extends PortalException {
+
+	public ThemeIdNotFoundStyleBookEntryFileException() {
+	}
+
+	public ThemeIdNotFoundStyleBookEntryFileException(String msg) {
+		super(msg);
+	}
+
+	public ThemeIdNotFoundStyleBookEntryFileException(
+		String msg, Throwable throwable) {
+
+		super(msg, throwable);
+	}
+
+	public ThemeIdNotFoundStyleBookEntryFileException(Throwable throwable) {
+		super(throwable);
+	}
```

Comment:
Do we need all these methods?

Reply:
No, I'll remove them

Reply:
I added back, just for consistency with the others